package.json

```diff
@@ -917,6 +917,27 @@
       },
       {
         "id": "sas-log"
+      },
+      {
+        "id": "sassql"
```

Comment:
Is there a chance to NOT declare a new language? Is it possible to just inject a textmate to source.sas?
https://code.visualstudio.com/api/language-extensions/syntax-highlight-guide#injection-grammars